File: haru-registry/src/main/java/haru/registry/HaruRegistry.java
package haru.registry;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer;

/**
 * @author takeshi
 */
@EnableEurekaServer
@SpringBootApplication
public class HaruRegistry {

  public static void main(String[] args) {
    SpringApplication.run(HaruRegistry.class, args);
  }

}

File: haru-registry/src/main/kotlin/haru/registry/HaruRegistry.kt
package haru.registry

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer

/**
 * @author takeshi
 */
@EnableEurekaServer
@SpringBootApplication
open class HaruRegistry {

  companion object {
    @JvmStatic
    fun main(args: Array<String>) {
      SpringApplication.run(HaruRegistry::class.java, *args)
    }
  }

}

